src/layout.rs

```diff
@@ -3335,12 +3395,12 @@ mod tests {
         FullscreenWindow(#[proptest(strategy = "1..=5usize")] usize),
         FocusColumnLeft,
         FocusColumnRight,
-        FocusWindowDown,
-        FocusWindowUp,
+        FocusWindowDown(#[proptest(strategy = "any::<bool>()")] bool),
```

Comment:
I don't think this is needed; the others specify a strategy just because they need something other than the default (i.e. numbers 1 through 5 instead of the whole usize range). For a bool it's not an issue.

Reply:
Ahhk, yea I definitely wasn't sure what I was doing there haha